api/src/main/java/net/kyori/adventure/text/format/TextDecoration.java

```diff
@@ -77,6 +77,17 @@
     this.name = name;
   }
 
+  /**
+   * Creates a {@link TextDecorationAndState}.
+   *
+   * @param state the state
+   * @return a {@link TextDecorationAndState}
+   * @since 4.8.0
+   */
+  public final @NonNull TextDecorationAndState as(final @NonNull State state) {
```

Comment:
Your example of `TextDecoration.BOLD.as(TextDecoration.State.TRUE)` from the tests is annoyingly verbose imo.

Does it make sense to add some helper methods like `asTrue()` and `asFalse()` to reduce that verbosity?

Reply:
`TextDecoration.BOLD.setTo(true)` reads a bit better IMO.

Reply:
`as(boolean)` added.